src/main/java/com/auth0/AuthorizeUrl.java

```diff
@@ -62,6 +63,23 @@ public AuthorizeUrl withConnection(String connection) {
         return this;
     }
 
+    /**
+     * Sets whether cookies used during the authentication flow have the {@code Secure} attribute set or not.
+     * By default, cookies will be set with the Secure attribute if the responseType includes {@code id_token} and thus requires
+     * the {@code SameSite=None} cookie attribute to set. Setting this to false will <strong>not</strong> override this behavior,
```

Comment:
It reads a bit odd. What about the following?
```suggestion
     * the {@code SameSite=None} cookie attribute set. Setting this to false will <strong>not</strong> override this behavior,
```